frontend/src/components/v1/StagedConfirmationTray.css

```
@@ -0,0 +1,250 @@
+.sct__step--done {
+  color: rgba(255, 255, 255, 0.5);
+}
+
+.sct__step-num {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: 1.4rem;
+  height: 1.4rem;
+  border-radius: 50%;
+  border: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)